pytest_splunk_addon/splunk.py

```diff
@@ -101,6 +101,13 @@ def pytest_addoption(parser):
             " of 8.0.0. "
         ),
     )
+    group.addoption(
+        "--dm-path",
```

Comment:
Can we update the param name to `--splunk-dm-path`?

Reply:
Done.